### PR TITLE
Api Usability

### PR DESCRIPTION
**Pull request by 36000**
As I work through using the API, I am putting any changes I make in this pull request. So far:
1. Allowing relative paths for dwi_folder, anat_folder (sub_dir is redundant as sess contains full path already)
2. Small additions to documentation
3. Using f strings for paths
4. Moved helper functions into the class
5. added force recompute

### CODE REVIEW

File: AFQ/api.py [hunk 1]
```diff
@@ -890,20 +894,17 @@ def __init__(self,
 
                 os.makedirs(results_dir_list[-1], exist_ok=True)
 
-                dwi_file_list.append(glob.glob(op.join(sub_dir,
-                                                       ('%s/%s/%s.nii.gz' %
-                                                        (sess, dwi_folder,
-                                                         dwi_file))))[0])
+                dwi_file_list.append(glob.glob('%s/%s/%s.nii.gz' %
```
Comment (36000) [inline AFQ/api.py]: Use f strings instead or path lib

**Comment by arokem** [comment]
Move all the helper functions to be methods of the AFQ object. 

Then, we can revisit `force_recompute` (see also #220)

**Comment by 36000** [comment]
How should force_recompute work? Should it be a parameter that the user can turn on or off at any time? Or an argument to the functions that the user can call?

**Comment by arokem** [comment]
> How should force_recompute work? Should it be a parameter that the user can turn on or off at any time? Or an argument to the functions that the user can call?

I think that it should be an attribute of the object, set to `False` per default. If the user sets this to `True`, everything that is already computed gets over-written.

**Comment by arokem** [comment]
This looks good!  @36000: Is it ready to merge, from your point of view?

**Comment by arokem** [comment]
Awesome. With this now merged, I am going to make another point release, to see that we get a sane version number on PyPI and that Zenodo uploads work.

**Pull request merged**